Jenkinsfile

```diff
@@ -183,9 +185,21 @@ pipeline {
                         }
                     }
                 }
+                stage('Mac Unit with Threading') {
```

Comment:
Suggestion: can we label these more explicitly now so it matches what we're saying? A more concrete suggestion:
`Unit: mac / clang 6 / libc++ Threading`.

Reply:
How do I know this is clang 6 and it uses libc++? Is there some way to specify that explicitly in the setup of make local?